components/Org/OrgDetail.js

```diff
@@ -107,70 +150,60 @@ const orgInstructionTab = (
-  <FullPage>
+  <OrgPageWrapper>
     <Head>
       <title>Voluntarily - {org.name}</title>
     </Head>
     <PageHeaderContainer />
     <ProfileContainer>
       <ProfileImage src={org.imgUrl} alt={org.name} />
       <TitleContainer>
-        <H3Bold>{org.name}</H3Bold>
+        <OrgHeader>{org.name}</OrgHeader>
+        <Markdown children={(org.info && org.info.about) || ''} />
+        <a href='{org.website}'>{org.website}</a>
+        {props.isAuthenticated && (
+          <RegisterMemberSection orgid={org._id} meid={props.me._id} />
+        )}
       </TitleContainer>
     </ProfileContainer>
     <ProfileContentContainer>
       <Tabs style={shadowStyle} defaultActiveKey='1' onChange={callback}>
         <TabPane tab={orgTab} key='1'>
           <TabContainer>
-            <Markdown children={(org.info && org.info.about) || ''} />
-            <OrgCategory orgCategory={org.category} />
-            <br />
-            <PBold>
-              <FormattedMessage
-                id='orgdetail.social.label'
-                defaultMessage='Social:'
-                description='Label for social media links on organisation details'
-              />
-            </PBold>
-            <ButtonGroup size='medium'>
-              {org.website && (
-                <SocialButton
-                  type='link'
-                  href={`${org.website}`}
-                  icon='global'
-                  target='_blank'
-                  rel='noopener noreferrer'
-                />
-              )}
-              {org.contactEmail && (
-                <SocialButton
-                  type='link'
-                  href={`mailto:${org.contactEmail}`}
-                  icon='mail'
-                  target='_blank'
-                  rel='noopener noreferrer'
-                />
-              )}
-              {org.facebook && (
-                <SocialButton
-                  type='link'
-                  href={`https://www.facebook.com/${org.facebook}`}
-                  icon='facebook'
-                  target='_blank'
-                  rel='noopener noreferrer'
-                />
-              )}
-              {org.twitter && (
-                <SocialButton
-                  type='link'
-                  href={`https://www.twitter.com/${org.twitter}`}
-                  icon='twitter'
-                  target='_blank'
-                  rel='noopener noreferrer'
-                />
-              )}
-            </ButtonGroup>
+            <h2>About</h2>
+            <Markdown style={{ fontSize: '1.5rem' }} children={(org.info && org.info.about) || ''} />
+          </TabContainer>
+          <Divider />
+          <TabContainer>
+            <h2>Contact</h2>
+            <ContactList>
+              <li>Website<br /><a href={org.website} target='_blank' rel='noopener noreferrer'>{org.website}</a></li>
+              <li>Twitter<br /><a href={org.twitter} target='_blank' rel='noopener noreferrer'>{org.twitter}</a></li>
```

Comment:
You have changed the behavior of the page so that the word Twitter and Facebook is shown even when the org does not have these links.   The previous code was correct and you do not need to change it.

Reply:
Sorry I have changed this back now.